parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -868,6 +868,24 @@ providers:
     apiVersion: credentialprovider.kubelet.k8s.io/v1
     args:
       - /etc/kubernetes/azure.json
+EOF
+    elif [[ -n "${BOOTSTRAP_PROFILE_CONTAINER_REGISTRY_SERVER}" ]]; then
```

Comment:
Since we have three if blocks, it's better to add logs for each of the path?

Reply:
added